.github/workflows/spell-checker.yml

```diff
@@ -0,0 +1,17 @@
+name: 📀 Spellchecker
+
+on: [pull_request]
+
+jobs:
+  misspell:
+    name: 🧹 Check Spelling
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🍉 Check Out
+        uses: actions/checkout@v2
+      - name: 🥥 Install
+        run: |
+          wget -O - -q https://git.io/misspell | sh -s -- -b .
+      - name: 🥦 Misspell
+        run: |
+          find docs README.md -type f | xargs ./misspell -error -i 'commandbox'
```

Comment:
Hey @zspitzer this has run against all docs and the only `tech term` it found that it didn't like was `commandbox` which I ignored.